scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```diff
@@ -57,7 +57,7 @@ spec:
             privileged: true  # Required for chcon to work on host files
       containers:
         - name: pccs
-          image: quay.io/openshift_sandboxed_containers/dcap/pccs:0.2.4
+          image: quay.io/redhat-user-workloads/ose-osc-tenant/osc-pccs@sha256:64c8df8575c0b9b67922a5b7adb0d14aed1637261a869ed4a341d58d4a16997b
```

Comment:
The changes look good to me. The only comment I have is that all other images are referring registry.redhat.io instead of quay.io. I wonder if that's because Mintmaker can update them. 

@littlejawa, looking at the references from quay.io, do you happen to know if it matters if these references are pointing to quay.io or registry.redhat.io?